dev-tools/compose/nginx_ingress.conf

```diff
@@ -63,7 +63,8 @@ server {
     location ^~ /dashboard {
         resolver 127.0.0.11 valid=30s;
         rewrite ^/dashboard(?!/)(.*)$ /dashboard/$1 permanent;
-        proxy_pass http://dashboard:5173;
+        set $dashboard_backend "http://dashboard:5173";
+        proxy_pass $dashboard_backend;
```

Comment:
this allows not bringing up protocol dashboard. previously if you tried to not bring it up, the nginx ingress container would exit because it can't resolve `http://dashboard` at init. by using a variable here, we defer DNS resolution to runtime